### DISCUSSION TITLE
2021.06.16

### DISCUSSION THREAD

**Discussion by HongzCloud** [malaheaven/issue-tracker#91] [General]
iOS: 
- 어제에 이어서 네트워크 관련 테스트 영상 시청 후 테스트 코드 작성했습니다.
- 키체인 테스트 코드 작성 했습니다. 
- 전에 로그인 했으면 바로 로그인된 화면으로 넘어가게 했습니다.
**iOS는 오늘 행복합니당** 😀

**Comment by malaheaven** [comment]
### BE
- domain 구현(Jpa 삽질 해결)
    - Jpa의 Entity클래스 구현을 조금 더 유연하게 적용하기 위한 강의 시청
- Request DTO, Response DTO 구현
- 코드에 변수명같은 부분을 통일하고자 API 수정 및 mock docs에 적용
    - [Mock API docs](https://documenter.getpostman.com/view/8052286/TzY7cDPK)

**BE도 오늘 행복합니다** 😄

**Comment by 17-sss** [comment]
### FE
- 어제에 이어서 이슈리스트에 쓰이는 Modal을 컨트롤 하는 로직을 리팩토링하였습니다
  - 너무 많은 삽질과 많은 방법을 적용했었고 다른 분들의 코드를 보며 학습하고 적용했습니다! (의미있던 하루인 것 같습니다!)
  - ~~이제 ListModal 컨트롤하는거 그만할래요..~~

**FE도 오늘 Happy 합니다 (●'◡'●) 😁**
